db/migrate/20161108105410_create_products.rb

```diff
@@ -0,0 +1,17 @@
+class CreateProducts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :products do |t|
+    	
+    	t.string :title
+    	t.boolean :taxable
+    	t.string :featured_image
+    	t.boolean :available
+    	t.string :price
+    	t.integer :grams
+    	t.string :compare_at_price
+    	t.bigint :product_id
```

Comment:
Naming this `product_id` might be a bit confusing (since we reference `Product` objects with `product_id` in other models). Perhaps something like `unique_product_id` might work better?